doc/classes/Object.xml

```diff
@@ -76,83 +76,48 @@
 		<method name="_get_property_list" qualifiers="virtual">
 			<return type="Dictionary[]" />
 			<description>
-				Override this method to customize how script properties should be handled by the engine.
+				Override this method to provide a custom list of additional properties to handle by the engine.
 				Should return a property list, as an [Array] of dictionaries. The result is added to the array of [method get_property_list], and should be formatted in the same way. Each [Dictionary] must at least contain the [code]name[/code] and [code]type[/code] entries.
+				You can use [method _property_can_revert] and [method _property_get_revert] to customize the default values of the properties added by this method.
 				The example below displays [code]hammer_type[/code] in the Inspector dock, only if [code]holding_hammer[/code] is [code]true[/code]:
```

Comment:
This needs to go with the example changed.

Reply:
You mean I should add it in the example?

Reply:
> The example below displays [code]hammer_type[/code] in the Inspector dock, only if [code]holding_hammer[/code] is [code]true[/code]:

But the example is different now, so the sentence is wrong.

Reply:
Ah right.